utils/led.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * led.h
+ *
+ *  Created on: 29-Jan-2021
+ *      Author: hp
+ */
+
+#ifndef UTILS_LED_H_
+#define UTILS_LED_H_
+
+void led_blink(led_error_type_t blinks);
+void led_set_high(void);
+
```

Comment:
I'd also add a `led_turn_off()` or `led_off()` here and implement it in `led.c`.

Reply:
Sure, done! Implemented `led_turn_off()` in `led.c`

Reply:
Still not solved yet, but I will resolve this and send "in-line" comments in the new code.